test-results/login-Login-Flow-should-show-error-on-invalid-login-chromium/error-context.md

```
@@ -0,0 +1,24 @@
+- Following Playwright test failed.
+- Explain why, be concise, respect Playwright best practices.
+- Provide a snippet of code with the fix, if possible.
+
+# Test info
+
+- Name: login.spec.ts >> Login Flow >> should show error on invalid login
+- Location: e2e\login.spec.ts:26:3
+
+# Error details
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)